### DISCUSSION TITLE
Export unified heading styles for all levels

### DISCUSSION THREAD

**Discussion by peter88213** [zim-desktop-wiki/zim-desktop-wiki#2425] [Ideas]
Is there an important reason to format the first two heading levels differently in Markdown export? 

**zim/formats/plain.py**
```
	def dump_h(self, tag, attrib, strings):
		# Markdown style headers
		level = int(attrib['level'])
		if level < 1:
			level = 1
		elif level > 5:
			level = 5

		if level in (1, 2):
			# setext-style headers for lvl 1 & 2
			if level == 1:
				char = '='
			else:
				char = '-'
			heading = ''.join(strings)
			underline = char * len(heading.strip('\n'))
			return [heading, underline + '\n']
		else:
			# atx-style headers for deeper levels
			tag = '#' * level
			strings.insert(0, tag + ' ')
			return strings
```

For further use of exported notes I would find a uniform formatting in atx style more practical.
This would make it possible to display horizontal rulers with the more commonly used three dashes instead of a row of asterisks, like here:

**zim/formats/markdown.py**

```
	def dump_line(self, tag, attrib, strings=None):
		return '\n{}\n'.format('*' * 5)
```

**Comment by hsandt** [comment]
An issue was opened: https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/469
